tests/multiparser_js_graphql/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -17,6 +17,8 @@ gql\`
 
 `;
 
+
+
```

Comment:
Just a curios Jest question: Does anybody know where these newlines came from?

Reply:
This got fixed when i merged with latest master.